lighthouse-core/report/html/html-report-assets.js

```diff
@@ -23,6 +23,9 @@ const REPORT_JAVASCRIPT = [
   fs.readFileSync(__dirname + '/renderer/pwa-category-renderer.js', 'utf8'),
   fs.readFileSync(__dirname + '/renderer/report-renderer.js', 'utf8'),
   fs.readFileSync(__dirname + '/renderer/i18n.js', 'utf8'),
+  // // TODO: minified version breaks utf8
```

Comment:
yikes! 😮  


aside, is there value leaving in as a TODO though? I thought we didn't really need it in report in first place. If it's a longer term, optimistic thing, perhaps an issue?